<?xml version="1.0"?>
<sdf version="1.9">
    <model name="grayscaled_camera">
        <link name="grayscaled_camera">
            <inertial>
                <mass>0.005</mass>
                <inertia>
                    <ixx>8.33e-06</ixx>
                    <ixy>0</ixy>
                    <ixz>0</ixz>
                    <iyy>8.33e-06</iyy>
                    <iyz>0</iyz>
                    <izz>8.33e-06</izz>
                </inertia>
            </inertial>
            <sensor name="camera" type="camera">
                <always_on>1</always_on>
                <update_rate>60</update_rate>
		  <camera>
		    <horizontal_fov>1.047</horizontal_fov>
		    <image>
		      <width>1280</width>
		      <height>720</height>
		      <format>L16</format> 
		    </image>
		    <clip>
		      <near>0.1</near>
		      <far>50</far>
		    </clip>
		  </camera>
             </sensor>
         </link>

        <frame name="mount_point"/>
    </model>
</sdf>
